### DISCUSSION TITLE
New Feature: Available Presets Wizard

### DISCUSSION THREAD

**Discussion by devajmeireles** [kool-dev/kool#373] [Ideas]
Hello, community!

I've been analyzing it and saw that it would be interesting maybe a wizard (or just a list) that demonstrates all the presets available in Kool. Maybe something like: `kool presets` (with no parameters) draw a wizard making available all Kool presets (or local presets too) for easy user selection.

Unfortunately I can't contribute with this feature because I don't know the Kool language (Go) :( But I leave a suggestion to the community. 

* I will try to study the basics of the language to see if I can contribute, if I can I will send a PR.

**Comment by danielsuguimoto** [comment]
For me, we can add this list to the `kool preset --help` output. Or we can add a subcommand `list` to the existing `preset` command.  But the wizard for preset already exists. If you run `kool preset`, some question will be prompt for you, guiding you to initialize a preset.
